### ISSUE TITLE
Games not starting with the Steam library on MergerFS

### ISSUE THREAD

**Issue by MasterCATZ** [ValveSoftware/steam-for-linux#6026]
#### Your system information

* Steam client version (build number or date): Jan 4 , 1544909276
* Distribution (e.g. Ubuntu): Ubuntu 18.04.1 LTS
* Opted into Steam client beta?: [Yes/No] Yes
* Have you checked for system updates?: [Yes/No] Yes

#### Please describe your issue in as much detail as possible:
Everything was working did no software updates only changed 
AMD based system to Intel Based system both were 64bit 

then I started getting wrong ELF class errors and 32bit errors , 
tried reinstalling steam same issue , 
purged steam and steam folders manually and reinstalled , same problems 

Tomb Raider  / Dirt Rally / Dues Ex Mankind Divided etc  
used to work now they silently crash 

[steamlog.txt](https://github.com/ValveSoftware/steam-for-linux/files/2767252/steamlog.txt)

using latest mesa drivers 
https://launchpad.net/~oibaf/+archive/ubuntu/graphics-drivers

Processor Information:
    CPU Vendor:  GenuineIntel
    CPU Brand:  Intel(R) Core(TM) i7-4790K CPU @ 4.00GHz
    CPU Family:  0x6
    CPU Model:  0x3c
    CPU Stepping:  0x3
    CPU Type:  0x0
    Speed:  4400 Mhz
    8 logical processors
    4 physical processors
    HyperThreading:  Supported
    FCMOV:  Supported
    SSE2:  Supported
    SSE3:  Supported
    SSSE3:  Supported
    SSE4a:  Unsupported
    SSE41:  Supported
    SSE42:  Supported
    AES:  Supported
    AVX:  Supported
    CMPXCHG16B:  Supported
    LAHF/SAHF:  Supported
    PrefetchW:  Unsupported

Operating System Version:
    Ubuntu 18.04.1 LTS (64 bit)
    Kernel Name:  Linux
    Kernel Version:  4.15.0-43-generic
    X Server Vendor:  The X.Org Foundation
    X Server Release:  11906000
    X Window Manager:  Compiz
    Steam Runtime Version:  steam-runtime-beta-release_2018-11-28

Video Card:
    Driver:  X.Org AMD HAWAII (DRM 2.50.0, 4.15.0-43-generic, LLVM 7.0.1)
    Driver Version:  4.5 (Compatibility Profile) Mesa 19.0.0-devel (git-929df7a 2019-01-14 bionic-oibaf-ppa)
    OpenGL Version: 4.5
    Desktop Color Depth: 24 bits per pixel
    Monitor Refresh Rate: 60 Hz
    VendorID:  0x1002
    DeviceID:  0x67b1
    Revision Not Detected
    Number of Monitors:  2
    Number of Logical Video Cards:  2
    Primary Display Resolution:  1280 x 1024
    Desktop Resolution: 2560 x 1024
    Primary Display Size: 14.80" x 11.85" (18.94" diag)
                                            37.6cm x 30.1cm (48.1cm diag)
    Primary VRAM: 4096 MB

Sound card:
    Audio device: Intel Haswell HDMI

Memory:
    RAM:  31513 Mb

Miscellaneous:
    UI Language:  English
    LANG:  en_AU.UTF-8
    Total Hard Disk Space Available:  210641 Mb
    Largest Free Hard Disk Block:  166018 Mb
    VR Headset: None detected

Recent Failure Reports:

**Comment by MasterCATZ** [comment]
the only game I can seem to get to run is Unreal Tournament 2004 that uses Direct3d,
unsure why all my other ones are not running 
I can not seem to get any form of log to show me what the issue is 
please any idea's what commands i can run ?

**Comment by MasterCATZ** [comment]
ok it seems steam no longer likes running things from mergerfs 

my fstab looks like this 

/mnt/SnapRaidShelves/* /SnapRaidShelves fuse.mergerfs comment=x-gvfs-show,defaults,allow_other,direct_io,use_ino,category.create=mfs,moveonenospc=true,minfreespace=20G,fsname=SnapRaidShelves 0 0

**Comment by kisak-valve** [comment]
Hello @MasterCATZ, are you still experiencing this issue on an up to date system?

**Comment by kisak-valve** [comment]
Closing pending feedback.

**Comment by MasterCATZ** [comment]
yes still not working via mergerfs file system I moved steam back to a ZFS file system

**Comment by zacjor** [comment]
Just wanted to say that I'm still experiencing issues with this. 2-3 weeks ago I was able to play most games with few issues on mergerfs, but as of around Monday 12-13-2021, all games on mergerfs fail to launch.
